api/message.ts

```diff
@@ -8,8 +8,19 @@ export enum Action {
   SIGN_AND_BROADCAST_TX,
   SIGN_TX,
   ETHEREUM_REQUEST,
+  SIGN_MESSAGE,
 }
 
+// ================================================================================================
+
+export const SignMessagePayloadSchema = z.object({
```

Comment:
This file is growing with schema. Maybe we can put them in the handlers directly.

You can also have schema directly inside a function (scope).